README.md

```diff
@@ -18,52 +33,38 @@ It's also ready to be used with [Codespaces](https://github.com/features/codespa
 
 ## For students and developers
 
-Thanks to Codespaces, you can work on your projects without having [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) installed locally.
-
-Using Codespaces you get [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) in the cloud, using a so called developer container. Like [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) installed locally, in this cloud version, you can install extensions, and use a terminal.
+Using Codespaces, you get [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza) in the cloud, using a ["developer container"](https://containers.dev/). Like a locally running version of [Visual Studio Code](https://visualstudio.microsoft.com/?WT.mc_id=academic-77460-alfredodeza), the cloud version also allows you to install extensions and use a terminal.
```

Comment:
Codespaces can be used with a local VSCode or a web-based VScode, but this paragraph changes that description. Linking to the Containers.dev site is a good idea, can we make it a link instead?

Reply:
containers.dev is currently a link, or do you mean that you want the link text to be containers.dev?
I didn't read the original version as implying that Codespaces could be used locally (I didn't actually know if that was the case). It defaults to cloud right?

Reply:
Will send a follow-up PR shortly! Also have some fixes now that I've gone through deployment process.